test/tc_allowed_syscalls_test.go

```diff
@@ -173,7 +173,7 @@ metadata:
 spec:
   containers:
   - name: test-container
-    image: quay.io/security-profiles-operator/test-nginx:1.19.1
+    image: nginxinc/nginx-unprivileged:1.21
```

Comment:
I assume that this might cause we hit the dockerhub rate limit. If this change is OK-ed in general, I can (if I have the perms?) push the image into our quay.io namespace.

Reply:
Yep, you already have the permissions to write to https://quay.io/organization/security-profiles-operator